pkg/logs/input/container/docker.go

```diff
@@ -198,7 +211,13 @@ func (dt *DockerTailer) forwardMessages() {
 		}
 		containerMsg := message.NewContainerMessage(updatedMsg)
 		msgOrigin := message.NewOrigin()
-		msgOrigin.LogSource = dt.source
```

Comment:
I think it's a good move to store only the config and not whole source, I just have one question: are we sure we won't need the source elsewhere downstream ?

Reply:
As far as we don't need to report any status downstream, we should be good.